src/panels/lovelace/elements/hui-icon-element.js

```diff
@@ -32,7 +33,13 @@ class HuiIconElement extends ElementClickMixin(PolymerElement) {
 
   ready() {
     super.ready();
-    this.registerMouse(this._config);
```

Comment:
I think we can get rid of the element click mixin now?

Reply:
It still handles actually picking and executing the actions, which could be done in the elements themselves, but that's some code duplication...

It's actually restored to *almost* what it was before the first long-press PR.

Reply:
nevermind saw you updated it already.